go.mod

```diff
@@ -19,6 +20,7 @@ require (
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/viper v1.7.0
```

Comment:
What are we using viper for?

Reply:
@edwarnicke if you have any idea what we can use else for monitoring file you can write it here https://github.com/networkservicemesh/sdk/issues/389 :)